common/proto/src/main/java/org/thingsboard/server/common/adaptor/JsonConverter.java

```diff
@@ -590,7 +590,8 @@ public static Map<Long, List<KvEntry>> convertToTelemetry(JsonElement jsonElemen
 
     public static Map<Long, List<KvEntry>> convertToSortedTelemetry(JsonElement jsonElement, long systemTs) throws
             JsonSyntaxException {
-        return convertToTelemetry(jsonElement, systemTs, true);
+        JsonElement timeseriesElement = jsonElement.isJsonObject() ? jsonElement.getAsJsonObject().get("timeseries") : null;
```

Comment:
I think we shouldn't modify JsonConverter. Rather split the processing of POST_TELEMETRY_REQUEST and TIMESERIES_UPDATED events in the DeviceState